docs/release-notes/release-notes-0.8.3.md

```diff
@@ -0,0 +1,253 @@
+
+An additional note: users who are more concerned with successful sweep transactions than a change in fee might be advised to bump the default from 0.8 to a much higher value like 2.0.
+
+`027e468` inverted feeratio so calculation of fee deviation works as intended
+
+### Auto-load the bitcoind wallet on startup if possible.
+
+See [PR 856](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/856). This is helpful now that the model of bitcoind RPC access to wallets has changed (note that there is no longer a "default wallet", and that wallets can be loaded dynamically. Read more about setting bitcoind RPC wallets [here](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/USAGE.md#setting-a-core-wallet-recommended) if this is new to you.
+
+`7f85822` Try load RPC wallet on start-up if it's not loaded
+
+### Cache RPC `gettransaction results for better performance
+
+See [PR 851](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/851). This should improve the performance of the `history` command by caching results from the RPC call.
+
+`6f533aa` Reduce number of gettransaction RPC's by caching results
+
+### More minor functional changes
+
+`ae57657` ob-watcher.py: gracefully exit on old python versions
+
+`7bfc758` Use mempoolminfee instead of minrelaytxfee as tx feerate floor
+
+`bc2d604` Prevent absurdly low minsizes in yg-pe
+
+`aab14f6` fix taker fee estimation on sweep after maker responses
+
+`1d74222` Fix year 2038 problem in fidelity bond wallets
+
+`8639a96` When selecting utxos in yg, count in that change should be above dust threshold already on the first try
+
+`43cc960` taker: Account for off-by-one errors caused by rounding
+
+`9ea0a73` Hide "Total balance" in wallet-tool when only single mixdepth is displayed
+
+`f361c98` Better handling of non-standard tx'es, less extra output for csv and other small improvements
+
+### Qt UI improvements
+
+These are mostly fairly minor so grouped together:
+
+`2bde2f9` jm-qt: Don't annoy the user for removing an address
+
+`1c2b6cd` jm-qt: Clarify error message: "0" is a valid input to `numCPInput`
+
+`4599bd7` Fix autofreeze warning on Qt
+
+`7949374` Clear single send form after successful send
+
+`25b2e8e` Don't create logs, wallets and cmtdata subdirectories under scripts
+
+
+### Testing and dev related background fixes
+
+These are almost entirely not related to user functionality, but a couple of notes:
+
+Although `df83764` is minor, it relates to an important point: we cannot support importing keys of a different scriptPubKey type (e.g. native segwit BIP84) than the wallet being imported into. Key import is not something users should generally use, but if they do, it's very important to know this limitation.
```

Comment:
```suggestion
Although `df83764` is minor, it relates to an important point: we cannot support importing keys of a different scriptPubKey type (e.g. native segwit BIP84) than the wallet being imported into. Key import is not something users should generally use, but if they do, it's very important to know this limitation.

```

Reply:
Is the delta only an extra newline? Hard to spot :)

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/912/commits/f1e440cd596e5d46d9da9d6bea0db9e1489d988d

Reply:
@AdamISZ Yes. :)